src/interaction/scroll.ts

```diff
@@ -20,9 +19,11 @@ function recompute(event: UIEvent = null): void {
     let x = Math.round((eventTarget as HTMLElement).scrollLeft);
     let y = Math.round((eventTarget as HTMLElement).scrollTop);
     let id = getId(eventTarget as Node);
-    target.observe(id);
     let current: ScrollData = {target: id, x, y, time: time()};
 
+    // Bail out if we are attempting to retrieve initial scroll position and it turns out to be (0,0)
```

Comment:
Comment isn't clear. Maybe something like 

We don't send any scroll events if the current position is at the top (0, 0). 

Also, what happen if the user scroll down and then back to 0,0. Wouldn't this change cause us to miss that case? I think this might need to be stateful -- on first event don't send 0,0 but send after that.

Reply:
I'll rewrite the comment. But the state is ensured by checking for null event. That will only be possible when we call it from the start function. In all other subsequent calls, event will not be null and therefore won't hit this clause.